maintenance/tracing/tracing_test.go

```diff
@@ -10,30 +10,25 @@ import (
 	"testing"
 
 	"github.com/opentracing/opentracing-go"
+	"github.com/stretchr/testify/require"
 
 	"github.com/gorilla/mux"
 )
 
-func TestHandlerIgnore(t *testing.T) {
-	r := mux.NewRouter()
-	r.Use(Handler("/test"))
-	r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {})
-
-	rec := httptest.NewRecorder()
-	req := httptest.NewRequest("GET", "/test", nil)
-
-	r.ServeHTTP(rec, req)
-}
```

Comment:
Is there any reason why you removed this test? I understand that there is no explicit validation in this test, but at least it will show, that the tracing does not panic. Maybe modify the test like you did with `TestHandler`.